### ISSUE TITLE
ImageNet training setup

### ISSUE THREAD

**Issue by resurgo97** [ychzhang/fracbnn#2]
Hi!
Your BNN model has good performance and I really appreciate your work.
I would like to reproduce your ImageNet experiment, but cannot find hyperparameters you used.
(i.e. number of blocks, learning rate, epochs, optimizer, ...)
Did you use the same training setup used on ReActNet paper?
Or, I would really appreciate if you upload the codes you used for ImageNet training.

Thanks!

**Comment by resurgo97** [comment]
Plus, I would like to know if you tried scaling the model for different sizes, or you only experimented with the proposed model size for fair comparison with the previous paper. Thanks.

**Comment by ychzhang** [comment]
Hi resurgo97,

The training setup and hyperparameters are discussed in Section 5.1 in the paper.
Plus, we've just released the official code and the pre-trained models for reproducing the results in the paper, including the ImageNet model. Please check the pointer updated in the Readme file.

We haven't tried to scale the model, but this is definitely an interesting thing to do.

Best,
Yichi
